docs/modules/integrations/pages/github-action/run-appium-script.adoc

```diff
@@ -0,0 +1,18 @@
+= Run Appium script on Kobiton devices
+:navtitle: Run Appium script
+
+Learn how to utilize Kobiton automation testing with Appium script and GitHub Action.
+
+NOTE: For more information, see link:https://github.com/marketplace/actions/kobiton-execute-appium-test[Official GitHub Marketplace Kobiton Action for Appium].
+
+== Before you start
+
+You'll need the following:
+
+* A Kobiton account. To create a Kobiton account visit link:https://portal.kobiton.com/register[Kobiton].

```

Comment:
Remove this line as it is kind of the same with the second bullet point.

Reply:
1st bullet point " A Kobiton account. To create a Kobiton account visit link:https://portal.kobiton.com/register[Kobiton]." removed.